src/libraries/System.IO.FileSystem/tests/Path/Exists.cs

```diff
@@ -0,0 +1,507 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Linq;
+using Xunit;
+
+namespace System.IO.Tests
+{
+    public class Path_Exists : FileSystemTest
```

Comment:
instead of writing all these tests, another approach might be to just reuse all the tests for File.Exists and Directory.Exists. Every place each of those tests asserts true, assert Path.Exists is true; every place they assert false, assert Path.Exists is false. Then you don't need to write any new tests.

Reply:
or both -- that way if ther'es a bug in Path.Exists, a Path test fails; but you also have high certainty that you match existing API behavior in any and all wierd variations.

Reply:
How about we move all the tests in ```File_Exists```  class that truly care only about file to an abstract ```FilePath_Exists``` class
and all the tests in ```Directory_Exists``` class that truly care only about directory to an abstract ```DirectoryPath_Exists``` class 

And then have ```File_Exists```, ```Directory_Exists```, ```Path_Exists``` have the implementation of ```Exists()``` method overriden in their own way.

Reply:
I've reused tests from both File and Directory classes